src/lib.rs

```diff
@@ -1153,12 +1155,6 @@ mod panic {
 
 struct LoggerGuard(&'static Log);
 
-impl Drop for LoggerGuard {
```

Comment:
What would the implementation of `Drop` synchronize on for `LoggerGuard`?  Is the idea to synchronize all `__log` calls around flush?   Is that how the need for `shutdown` is removed?